response.go

```
@@ -0,0 +1,44 @@
+}
+
+func (r BaseResponse) GetContentType(c *gin.Context) string {
+	return r.ContentType
+}
+
+func (r BaseResponse) SetCode(code int) {
+	r.Code = code
+}
+
+func (r BaseResponse) SetContent(content any) {
+	r.Content = content
```

golangci-lint (1.64.8)

unusedwrite: unused write to field Content

(govet)